internal/cmd/common.go

```diff
@@ -26,6 +27,13 @@ func readMarkdownFile(args []string) ([]byte, error) {
 	if arg == "" {
 		f, err := os.DirFS(fChdir).Open(fFileName)
 		if err != nil {
+			if pathErr, ok := err.(*os.PathError); ok {
+				switch pathErr.Err {
+				case syscall.ENOENT, os.ErrNotExist:
```

Comment:
`ENOENT` and `ErrNotExist` are [the same errors](https://github.com/golang/go/blob/25f5d9d4a253cb880bff909ebbdc05c8941c4a48/src/syscall/syscall_unix.go#L123-L124). It's very unusual to compare errors returned by stdlib with `syscall` errors.

My suggestion is as follows:

```go
                if err != nil {
-                       return nil, errors.Wrapf(err, "failed to read %s", filepath.Join(fChdir, fFileName))
+                       var pathError *os.PathError
+                       if errors.As(err, &pathError) {
+                               return nil, errors.Errorf("failed to %s markdown file %s: %s", pathError.Op, pathError.Path, pathError.Err.Error())
+                       } else {
+                               return nil, errors.Wrapf(err, "failed to read %s", filepath.Join(fChdir, fFileName))
+                       }
                }
```

Which will produce something like:

```
./runme --filename nonexistent.md ls
failed to open markdown file /path/to/nonexistent.md: no such file or directory
```